spaceship/spec/connect_api/token_spec.rb

```diff
@@ -215,10 +215,12 @@
       payload, header = JWT.decode(token.text, public_key, true, { algorithm: 'ES256' })
 
       expect(payload['iss']).to eq(issuer_id)
+      expect(payload['iat']).to eq(Time.now.to_i)
```

Comment:
@nnsnodnb Excuse me if I'm asking a silly thing, but I'm wondering if Time.now is going to be exactly the same value as the one set in the token creation time. Is not there any chance to get a race condition here?